src/components/ModalContainer.scss

```diff
@@ -1,14 +1,10 @@
-@import '@edx/brand/paragon/variables.scss';
-@import '@openedx/paragon/scss/core/core.scss';
-@import '@edx/brand/paragon/overrides.scss';
-
 .ora-modal-body {
-  min-height: calc(100vh - 20px - 64px - (map-get($spacers, 2) * 8));
+  min-height: calc(100vh - 20px - 64px - calc(var(--pgn-spacing-spacer-3) * 8));
```

Comment:
Looks like this was 2 (`map-get($spacers, 2`) before and now it's 3 `calc(var(--pgn-spacing-spacer-3)` - is that intentional?

Reply:
By mistake, fixed! Thanks